modules/distr-tracing-rn-known-issues.adoc

```diff
@@ -44,8 +44,6 @@ These are the known issues for {DTProductName}:
 
 
 
-* link:https://issues.redhat.com/browse/OBSDA-220[OBSDA-220] In some cases, if you try to pull an image using {OTELShortName}, the image pull fails and a `Failed to pull image` error message appears. 
-There is no workaround for this issue.
 
 * link:https://issues.redhat.com/browse/TRACING-2057[TRACING-2057] The Kafka API has been updated to `v1beta2` to support the Strimzi Kafka Operator 0.23.0. However, this API version is not supported by AMQ Streams 1.6.3. If you have the following environment, your Jaeger services will not be upgraded, and you cannot create new Jaeger services or modify existing Jaeger services:
```

Comment:
```suggestion
* The Kafka API has been updated to `v1beta2` to support the Strimzi Kafka Operator 0.23.0. However, this API version is not supported by AMQ Streams 1.6.3. If you have the following environment, your Jaeger services will not be upgraded, and you cannot create new Jaeger services or modify existing Jaeger services:
// link:https://issues.redhat.com/browse/TRACING-2057[TRACING-2057]
```

Reply:
I will leave this out as it was not subject of this PR. There are probably other links in the changelog that would need to be removed.